static/fluent/de/main.ftl

```diff
@@ -0,0 +1,526 @@
+  .value = Abbrechen
+rules-delete-confirm =
+  .value = Regel löschen
+rule-invalid = Ungültig
+rule-delete-prompt = Soll die Regel dauerhaft gelöscht werden?
+rule-delete-cancel-button =
+  .value = Abbrechen
+rule-delete-confirm-button =
+  .value = Regel löschen
+rule-select-property = Merkmal wählen
+rule-not = Nicht
+rule-event = Ereignis
+rule-action = Aktion
+rule-configure = Konfigurieren...
+rule-time-title = Tageszeit
+rule-notification = Benachrichtigung
+notification-title = Titel
+notification-message = Nachricht
+notification-level = Level
+notification-low = Niedrig
+notification-normal = Normal
+notification-high = Hoch
+rule-name = Regelname
+rule-and = UND
+rule-or = ODER
+rule-time-of-day-is = die Tageszeit ist
+rule-event-lower = Ereignis
+rule-event-occurs = Vorkommen
+
+## Logs
+logs = Protokolle
+logs-create-log-hint = Keine Protokolle vorhanden. Für ein neues Protokoll + drücken.
+logs-device = Gerät
+logs-device-select =
+  .aria-label = Gerät protokollieren
+logs-property = Merkmal
+logs-property-select =
+  .aria-label = Merkmal protokollieren
+logs-retention = Speicherung
+logs-retention-length =
+  .aria-label = Speicherdauer
+logs-retention-unit =
+  .aria-label = Einheit der Speicherdauer
+logs-hours = Stunden
+logs-days = Tage
+logs-weeks = Wochen
+logs-save = Speichern
+logs-remove-dialog-title = Löschen
+logs-remove-dialog-warning = Beim Löschen eines Protokolls gehen alle bislang gesammelten Daten verloren.
+  Soll das Protokoll wirklich gelöscht werden?
+logs-remove = Löschen
+logs-unable-to-create = Das Protokoll konnte nicht angelegt werden
+logs-server-remove-error = Serverfehler: Das Protokoll konnte nicht entfernt werden
+
+## Add New Things
+add-thing-scanning-icon =
+  .alt = Suche
+add-thing-scanning = Es wird nach neuen Geräten gesucht...
+add-thing-add-adapters-hint = Keine neuen Geräte gefunden. 
+add-thing-add-adapters-hint-anchor = Neue Erweiterungen hinzufügen.
+add-thing-add-by-url = Via URL hinzufügen...
+add-thing-done = Erledigt
+add-thing-cancel = Abbrechen
+
+## Context Menu
+context-menu-choose-icon = Bild wählen...
+context-menu-save = Speichern
+context-menu-remove = Löschen
+
+## Capabilities
+OnOffSwitch = Ein/Aus Schalter
+MultiLevelSwitch = Mehrfachschalter
+ColorControl = Farbsteuerung
+EnergyMonitor = Energie-Monitor
+BinarySensor = Binärer Sensor
+MultiLevelSensor = Mehrfachssensor
+SmartPlug = Zwischenstecker
+Light = Licht
+DoorSensor = Türsensor
+MotionSensor = Bewegungssensor
+LeakSensor = Wassersensor
+PushButton = Schalter
+VideoCamera = Videokamera
+Camera = Kamera
+TemperatureSensor = Temperatursensor
+Alarm = Alarmanlage
+Thermostat = Thermostat
+Lock = Schloss
+Custom = Benutzerdefiniertes Gerät
+
+## Properties
+alarm = Alarm
+pushed = Gedrückt
+not-pushed = Nicht gedrückt
+on-off = Ein/Aus
+on = Ein
+off = Aus
+power = Strom
+voltage = Spannung
+temperature = Temperatur
+current = Aktuell
+frequency = Frequenz
+color = Farbe
+brightness = Helligkeit
+leak = Leck
```

Comment:
Without checking the english version i think we need an adjective here, like "Undicht"/"Feucht"